### PR TITLE
Disable recommendation feature for DVO specified in BDD

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Disable recommendation feature for DVO specified in BDD

## Type of change

- New test scenario added into existing feature file

## Testing steps

N/A

## Checklist
* [ ] Pylint passes for Python sources
* [ ] sources has been pre-processed by Black
* [ ] updated documentation wherever necessary
* [ ] new tests can be executed both locally and within docker container

### CODE REVIEW

**Review by Bee-lee — APPROVED** [review]
ok

File: features/DVO_Recommendations/namespace_disable.feature [hunk 1]
```diff
@@ -3,3 +3,151 @@ Feature: it is expected that customers will see some false positives for
...
+          | Downloads                | no                         |
+          | Advisor                  | yes                        |
+          | Vulnerability Dashboard  | no                         |
+          | Subscriptions            | no                         |
+          | Cost Management          | no                         |
+          | Support Cases            | no                         |
+          | Cluster Manager Feedback | no                         |
+          | Red Hat Marketplace      | no                         |
+          | Documentation            | no                         |
+     When user expand "Advisor" top level item
```
Comment (juandspy) [inline features/DVO_Recommendations/namespace_disable.feature:40]: expands

File: features/DVO_Recommendations/namespace_disable.feature [hunk 2]
```diff
@@ -3,3 +3,151 @@ Feature: it is expected that customers will see some false positives for
...
+     Then the menu should be expanded under "Advisor" top level item
+      And following new items should be displayed in the sub-menu on the left side as can be seen on https://github.com/RedHatInsights/insights-behavioral-spec/blob/main/ui/hcc_workloads_menu_item.png
+          | Expanded menu item  | Required for this scenario |
+          | Clusters            | no                         |
+          | Recommendations     | no                         |
+          | Workloads           | yes                        |
+     When user select "Workloads" menu item from this sub-menu
+     Then an "Workloads" page should be displayed right of the left menu bar
```
Comment (juandspy) [inline features/DVO_Recommendations/namespace_disable.feature:48]: an -> a

**Review by juandspy — APPROVED** [review]

File: features/DVO_Recommendations/namespace_disable.feature (hunk 1 above)
Comment (tisnik) [reply features/DVO_Recommendations/namespace_disable.feature]: fixed

File: features/DVO_Recommendations/namespace_disable.feature (hunk 2 above)
Comment (tisnik) [reply features/DVO_Recommendations/namespace_disable.feature]: fixed

**Pull request merged**